src/geo/points.ts

```diff
@@ -72,7 +79,34 @@ const distance = (pointFrom: Point, pointTo: Point): number | null => {
   return earthRadius * c
 }
 
+const equals = (point1: Point, point2: Point): boolean => {
+  if (point1 === null && point2 === null) return true
+  if (point1 === null || point2 === null) return false
+  return point1.srs === point2.srs && point1.x === point2.x && point1.y === point2.y
+}
+
+const isValid = (point: Point): boolean => {
+  if (!point || !isFilled(point)) return false
+
+  try {
+    SRSs.getSRSByCode(point.srs)
```

Comment:
should this return undefined instead of throwing an exception ?

Reply:
then you have to handle SRS | undefined values every time you use it.
otherwise we could add an "exists" function to check if it exists before calling getSRSByCode

Reply:
maybe it's simpler, I'll return undefined, ok